platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/CMakeLists.txt

```diff
@@ -9,19 +9,13 @@ if("TFM" IN_LIST MBED_TARGET_LABELS)
     add_subdirectory(TARGET_TFM)
 endif()
 
-target_include_directories(mbed-psa
+target_include_directories(mbed-core
     INTERFACE
-        ./inc
-        ./inc/psa
+        inc
+        inc/psa
 )
 
-target_sources(mbed-psa
+target_sources(mbed-core
     INTERFACE
         src/psa_hrng.c
 )
-
-target_link_libraries(mbed-psa
-    INTERFACE
-        mbed-mbedtls
```

Comment:
These 2 dependencies are missing in this pull request. The latest failure shows mbedtls must be included.

Reply:
PSA was made part of `mbed-core` because some mbed-core code depended on it. Its use is gated by the presence of the `EXPERIMENTAL_API` and `PSA` labels.